CMakeLists.txt

```diff
@@ -10,8 +10,13 @@ if(NOT CMAKE_CXX_STANDARD)
   set(CMAKE_CXX_STANDARD 14)
 endif()
 
+# Set the build type to Debug if not specified
+if(NOT CMAKE_BUILD_TYPE)
+  set(CMAKE_BUILD_TYPE Debug)
+endif()
```

Comment:
I think you should try with these lines removed. By default most build generator will generate a debug configuration, though the `CMAKE_BUILD_TYPE` variable is empty.